agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/statsbeat/CustomDimensions.java

```diff
@@ -63,7 +63,7 @@
     }
 
     sdkVersion = qualifiedSdkVersion.substring(qualifiedSdkVersion.lastIndexOf(':') + 1);
-    runtimeVersion = System.getProperty("java.version");
+    runtimeVersion = System.getProperty("java.specification.version");
```

Comment:
is this intentional?

Reply:
No. not sure why this kept popping up.. i rebased that commit already.. will remove it..